README.rst

```diff
@@ -58,12 +58,178 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+In order to use the **nidaqmx** package, you must have at least one DAQ (`Data Acquisition <https://www.ni.com/en/shop/data-acquisition.html>`_)
+device installed on your system. Both physical and simulated devices are supported. The examples below use an X Series DAQ device
+(e.g.: PXIe-6363, PCIe-6363, or USB-6363).
+You can use **NI MAX** or **NI Hardware Configuration Utility** to verify and configure your devices.
+
+Finding and configuring device name in **NI MAX**:
+
+.. image:: https://raw.githubusercontent.com/ni/nidaqmx-python/ca9b8554e351a45172a3490a4716a52d8af6e95e/max_device_name.png
+  :alt: NI MAX Device Name
+  :align: center
+  :width: 800px
+
+
+Finding and configuring device name in **NI Hardware Configuration Utility**:
+
+.. image:: https://raw.githubusercontent.com/ni/nidaqmx-python/ca9b8554e351a45172a3490a4716a52d8af6e95e/hwcu_device_name.png
+  :alt: NI HWCU Device Name
+  :align: center
+  :width: 800px
+
+Tasks in NI-DAQmx
+-----------------
+A task is a collection of one or more virtual channels with timing, triggering, and other properties.
+Refer to `NI-DAQmx Task <https://www.ni.com/docs/en-US/bundle/ni-daqmx/page/tasksnidaqmx.html>`_ for more information.
+
+Example code to create a task:
+
+.. code-block:: python
+
+  >>> import nidaqmx
+  >>> with nidaqmx.Task() as task:
+  ...     pass
+
+Virtual Channels in NI-DAQmx
+----------------------------
+Virtual channels, or sometimes referred to generically as channels, are software entities that encapsulate the physical channel
+along with other channel specific information (e.g.: range, terminal configuration, and custom scaling) that formats the data.
+A physical channel is a terminal or pin at which you can measure or generate an analog or digital signal.A single physical channel
+can include more than one terminal, as in the case of a differential analog input channel or a digital port of eight lines.
+Every physical channel on a device has a unique name (for instance, SC1Mod4/ai0, Dev2/ao5, and Dev6/ctr3) that follows the
+NI-DAQmx physical channel naming convention.
+Refer to `NI-DAQmx Channel <https://www.ni.com/docs/en-US/bundle/ni-daqmx/page/chans.html>`_ for more information.
+
+Example code that adds an analog input channel to a task, configures the range, and reads data.
```

Comment:
Looking at other subsections, I think this should be a colon instead of full stop?
```suggestion
Example code that adds an analog input channel to a task, configures the range, and reads data:
```

Reply:
good catch.